pkg/kubelet/dockertools/manager.go

```diff
@@ -62,6 +62,8 @@ const (
 
 	MinimumDockerAPIVersion = "1.18"
 
+	dockerv110APIVersion = "1.21"
```

Comment:
nit: Did you rebase? https://github.com/kubernetes/kubernetes/pull/21790/files already introduced version check for docker v1.10